src/vsr/replica.zig

```diff
@@ -4303,11 +4334,14 @@ pub fn ReplicaType(
             assert(op <= self.op);
             assert((op + 1) % constants.lsm_compaction_ops == 0);
             log.info("{}: commit_checkpoint_data: checkpoint_data start " ++
-                "(op={} current_checkpoint={} next_checkpoint={})", .{
+                "(op={} current_checkpoint={} next_checkpoint={} free_set.acquired={} " ++
+                "free_set.released={}", .{
```

Comment:
(Missing close-paren, and also may as well balance the lines a little.)
```suggestion
                "(op={} current_checkpoint={} next_checkpoint={} " ++
                "free_set.acquired={} free_set.released={})", .{
```